en-US/M.xml

```diff
@@ -219,6 +219,27 @@
 
 			</listitem>
 
+		</varlistentry>
+		 <varlistentry id="mission-critical">
+			<term>mission-critical</term>
+			 <listitem>
+				<para>
+					<emphasis>adj.</emphasis> Write hyphenated. 
+				</para>
+				<para>
+					Use to refer to software or systems that are essential to fulfilling the core purpose of an organization. 
+					When mission-critical systems fail, the impact on the organization is often immediate and always severe. 
```

Comment:
Consider removing "on the organization". It's not wrong, but the immediate impact might be felt elsewhere. Consider the example of flight tracking software. Yes, that would impact the org, but it would impact planes and passengers first, and worst. wdyt?